README.rst

```diff
@@ -107,16 +116,74 @@ You'll find six directories inside the main
 Running Simulations
 -------------------
 
-Before running a simulation, you should have a model specification file.
-A model specification is a complete description of a vivarium model in
-a yaml format.  An example model specification is provided with this repository
-in the ``model_specifications`` directory.
+To run this simulation, the first step is to generate an artifact with base GBD data in it.
+This will only work on the IHME cluster, because it pulls draw-level data from internal GBD databases.:::
+
+  :~$ conda activate vivarium_gates_mncnh_artifact
+  (vivarium_gates_mncnh_artifact) :~$ make_artifacts -l "Pakistan" -o artifacts/
+
+Replace "Pakistan" with the name of the location of interest.
+Only Pakistan, Nigeria, and Ethiopia are supported currently.
+This command will create an artifact file in the ``artifacts/`` directory within the repo;
+omit the ``-o`` argument to output to the default location of ``/mnt/team/simulation_science/pub/models/vivarium_gates_mncnh/artifacts``,
+or change to a different path.
+
+.. todo::
 
-With this model specification file and your conda environment active, you can then run simulations by, e.g.::
+  Include how to re-calculate LBWSG RR caps
```

Comment:
There are instructions in [generation script](https://github.com/ihmeuw/vivarium_gates_mncnh/blob/main/src/vivarium_gates_mncnh/data/lbwsg_rr_caps/generate_caps.py) as well if that helps (although I noticed the default output directory is wrong - I'll be fixing that in my next PR).